.aspect/workflows/bazelrc

```diff
@@ -1,3 +1,3 @@
 # build without the bytes
-common --remote_download_minimal
-common --nobuild_runfile_links
+# common --remote_download_minimal
+# common --nobuild_runfile_links
```

Comment:
Intentional?

Reply:
Temporary to green it up. Planning to deal with "build without the bytes" + delivery in a follow-up.